### ISSUE TITLE
Detailed documentation and data

### ISSUE THREAD

**Issue by hxue3** [adf1178/QueCos_code#1]
Hi, I am trying to use your code. However, I noticed this repo is not a compeleted version, as training data is missing. 

Is there a more detailed dcoumentation how to use your code? And if you can share the training dataset as well.

**Comment by adf1178** [comment]
Hi~
Sorry for the unclear documentation of the code ... the implementation of QueCos is very chaotic but I think the code is completed. The main part of QueCos is in src/ke, where it connects different code bases UNIF, DeepCS, OCoR and CodeBERT. Firstly, you can train a code search model such as UNIF and put the trained model in the corresponding dir. Then modify the argument --sent_reward to use pre-trained  UNIF model to compute the reward.  In `src/ke/lib/metric/RetReward.py` we implement the reward computation function for four code search models. 
As for data, I am so sorry there is something wrong when we crawled the data such as repetition :( if you want training data to run the script, we can send it to you.

**Comment by Ifra25** [comment]
Hi, can you please share the training data also.

**Comment by xiangzheng666** [comment]
Hi, can you please share the training data also.

**Comment by SIPUNK** [comment]
Hi, can you please share the training data also.

**Comment by chengong12** [comment]
Hi, can you please share the training data also.

**Comment by googol116** [comment]
Hi, can you please share the training data also.
